modules/sharedtest/testconstants/names.go

```diff
@@ -1,42 +1,23 @@
 package testconstants
 
 import (
-	"hash/fnv"
-	"strconv"
+	"math/rand"
 	"strings"
-
-	. "github.com/onsi/ginkgo/v2"
 )
 
-func fiveDigitTestHash(s string) string {
-	bitCount := 5 * 4
-	clearBitCount := 32 - bitCount
-	h := fnv.New64()
-	_, _ = h.Write([]byte(s))
-
-	hash := h.Sum64()
-	suiteConfig, _ := GinkgoConfiguration()
-	// mix with the seed and node number
-	hash = hash ^ uint64(suiteConfig.RandomSeed+int64(suiteConfig.ParallelProcess))
-
-	// xor lower 32 bits with higher 32 bits and keep only lower bits
-	hash32 := uint32(((hash ^ (hash >> 32)) << 32) >> 32)
-
-	// xor highest 12 bits with lower ones and keep only lower bits
-	hash32 = ((hash32 ^ (hash32 >> bitCount)) << clearBitCount) >> clearBitCount
+const chars = "abcdefghijklmnopqrstuvwxyz1234567890"
 
-	// forcefully add left fifth digit if lower number
-	lowest5DigitNum := uint32(1) << (bitCount - 4)
-	if hash32 < lowest5DigitNum {
-		hash32 ^= lowest5DigitNum
+func fiveDigitTestHash() string {
```

Comment:
To generate a random string, you can use the `rand.String()` function from `k8s.io/apimachinery/pkg/util/rand` package.

The `k8s.io/apimachinery` is already imported in this `sharedtest` package.

Reply:
fixed